build.rs

```diff
@@ -123,6 +123,14 @@ fn static_ruby_location() -> String {
     location
 }
 
+#[cfg(not(target_os = "macos"))]
+fn macos_static_ruby_dep() {}
+
+#[cfg(target_os = "macos")]
+fn macos_static_ruby_dep() {
+    println!("cargo:rustc-link-lib=framework=Foundation");
```

Comment:
This is required when linking the static ruby lib, as defined in `RbConfig::CONFIG`

```
"LIBRUBYARG_STATIC"=>"-lruby.2.5.3-static -framework Foundation"
```